### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by roncodeh** [ExpressLRS/ExpressLRS#1746] [Q&A]
I keep getting this
![16596876070365380581260628478705](https://user-images.githubusercontent.com/104334134/183034613-f4ba6c7c-3ac5-4bd2-a143-83446941e561.jpg)

**Comment by deadbytefpv** [comment]
Flash via WiFi (Build firmware and Upload manually) and when it say on the Popup "Target Mismatch", click on "Flash Anyway".

**Comment by roncodeh** [comment]
It's a timeout and a failed status too

**Reply by deadbytefpv** [reply]
if it's timing out, then the wifi is disconnecting, or that the receiver is rebooting due to insufficient power.
How is it being powered up?

Your other option is to put the receiver into bootloader mode manually and reflash via passthrough.

**Comment by roncodeh** [comment]
How do I input a response yes to the wrong target message?

**Reply by deadbytefpv** [reply]
You can't.

This is only available in the platformIO Build commands.

**Comment by roncodeh** [comment]
I am pretty sure that the rx serial is correctly set in betaflight.

So it is definitely the right selections in the ELRS configurator itself.

The passthrough worked but after it resets to bootloader it just says wrong
target.

Does this mean that my RX is a BETAFPV rx? Or is it an error?

**Reply by deadbytefpv** [reply]
This is happening because BetaFPV has used the wrong target from factory. the BetaFPV 2400 RX target is ONLY for their NANO receivers because it has a PA/LNA chip, while the Lite receivers doesn't have it, and thus, it is like a Generic receiver, like the EP1 or EP2, which were based off the DIY receivers.

**Comment by roncodeh** [comment]
Thank you so much! Still trying to make my drone fly. Everything works and
betaflight registers throttle and yaw inputs but nothing happens when I
unplug it and power on my drone and TX12

**Reply by deadbytefpv** [reply]
Try connecting the receiver into a different uart if it only works with usb plugged in. 
The UART where it's connected right now might have a pull-down resistor that only activates with LiPo plugged in.
